go.mod

```diff
@@ -334,8 +334,8 @@ replace (
 	github.com/mattn/go-shellwords => github.com/mattn/go-shellwords v1.0.5
 	github.com/matttproud/golang_protobuf_extensions => github.com/matttproud/golang_protobuf_extensions v1.0.1
 	github.com/mesos/mesos-go => github.com/mesos/mesos-go v0.0.9
-	github.com/mholt/caddy => github.com/mholt/caddy v1.0.0
-	github.com/mholt/certmagic => github.com/mholt/certmagic v0.5.0
+	github.com/mholt/caddy => github.com/caddyserver/caddy v1.0.1
```

Comment:
this line should change to `github.com/caddyserver/caddy => github.com/caddyserver/caddy v1.0.1`